README.md

```diff
@@ -1,4 +1,4 @@
-# ChaosBot
```

Comment:
Did I mess it up? I had a created a fake PR for testing that chained the title in the README to `ChaosBoat`. Except my instance of chaosbot actually merged it, so I reverted it.